src/manageiq-validators/manageIqPropTypes.js

```diff
@@ -0,0 +1,15 @@
+import { emailPattern } from './';
+
+export const emailPropType = ({
+  props,
+  propName,
+  componentName,
+  isRequired,
+}) => {
+  if (!props[propName] && !isRequired) {
+    return undefined;
+  }
+  return emailPattern.test(props[propName])
+    ? undefined
+    : new Error(`Invalid prop  ${propName} supplied to ${componentName} Validation failed. Expected email address.`);
```

Comment:
Maybe use the `__('bla')__`

Reply:
Oh you mean for the error messages. Nah it will be visible only in console, this is a message for developers not users. Its just a custom prop error message.